docs/index.rst

```diff
@@ -9,14 +9,30 @@ automation-tools documentation
 A set of tools to help automating virtual machines to install Foreman and test
 it using `Robottelo <https://github.com/SatelliteQE/robottelo>`_.
 
-Usage examples
-==============
+Installation
+============
+
+Install the following packages on your Operating System.
+
+- libxml
+- libxml-devel
```

Comment:
libxml-devel and libxslt-devel are the required ones and will take care to install other as needed.

You be good to put a sample installation command for Red Hat based systems

```
yum -y install libxml-devel libxslt-devel
```